pages/common/git-check-ignore.md

```diff
@@ -0,0 +1,24 @@
+# git check-ignore
+
+> Analyse and debug Git ignore / exclude (".gitignore") files.
+> More information: <https://git-scm.com/docs/git-check-ignore>.
+
+- Check whether a file or directory is ignored:
+
+`git check-ignore {{path/to/file_or_directory}}`
+
+- Check whether multiple files or directories are ignored:
+
+`git check-ignore {{path/to/file}} {{path/to/directory}}`
+
+- Use pathnames, one per line, from stdin:
```

Comment:
I'd suggest using either STDIN or `stdin`.

Reply:
I think we've previously chosen to [lowercase stdin in the majority of pages](https://github.com/tldr-pages/tldr/search?q=stdin). But I'll update it to use backticks.

Reply:
@pxgamer make sense. Let's merge this one as-is and standardize `stdin`/`stdout` in a new PR :)

Reply:
(I'm working on that, btw)

Reply:
Update: #3114